### PR TITLE
Add Czech translation (provided by @Flippo008)

### LINKED ISSUE #339: Czech translation
Hi, 
I read #313 and my case is totally the same. I have been learning Ultimate geography and making my own Czech translation because for me is crucial to know the names of countries etc. in my language. After v4.0 release I am planning to make a clean import and start from a scratch since the translation wasn't as good as I would like to be for public sharing. Same as hoi3 I don't know how to proceed correctly and I also read your answer that the maximum number of translation was reached. But I will make the Czech translation for my purpose regardless of sharing the translation to other people, so if the work could be incorporated later in the deck, that would be just a sweet bonus for me.

Sorry for eventual mistakes in this issue, obviously I am not a native speaker. 

I am looking forward to v4.0 of Ultimate geography.

### PR DESCRIPTION
**Pull request by aplaice**
Fix #339.

(I don't speak Czech, but from a general pattern-matching point-of-view, the translations look great! I think that I've also now updated everything else that needs updating.)

### CODE REVIEW

**Comment by aplaice** [comment]
I can squash the changes — splitting everything into separate commits helped me make sure that I wasn't messing anything up...

File: recipes/legacy_csv_from_split_csvs.yaml [hunk 1]
```diff
@@ -25,39 +25,45 @@
 
         - name: Country
         - name: Country:es
+        - name: Country:de
```
Comment (ohare93) [inline recipes/legacy_csv_from_split_csvs.yaml:28]: Should this not be above Spanish? As the rest below are.

Comment (aplaice) [reply recipes/legacy_csv_from_split_csvs.yaml:27]: Oops, yes!

File: src/data/capital.csv [hunk 2]
```diff
@@ -1,222 +1,222 @@
...
+Moldova,Chișinău,Chișinău,Chisináu,Chișinău,Chisinau,Kišiněv
+Monaco,Monaco,Monaco,Mónaco,Monaco,Monaco,Monaco-Ville
+Montenegro,Podgorica,Podgorica,"Podgorica, Cetiña",Podgorica,Podgorica,Podgorica
+Netherlands,Amsterdam,Amsterdam,Ámsterdam,Amsterdam,Amsterdam,Amsterdam
+Norway,Oslo,Oslo,Oslo,Oslo,Oslo,Oslo
+Poland,Warsaw,Warschau,Varsovia,Varsovie,Warszawa,Varšava
+Portugal,Lisbon,Lissabon,Lisboa,Lisbonne,Lisboa,Lisabon
+Romania,Bucharest,Bukarest,Bucarest,Bucarest,Bucuresti,Bukurešť
+Russia,Moscow,Moskau,Moscú,Moscou,Moskva,Moskva
+San Marino,City of San Marino,Stadt San Marino,San Marino,Ville de Saint-Marin,San Marino by,San Marino (město)
```
Comment (axelboc) [inline src/data/capital.csv:45]: I've spotted a few occurrences of disambiguation texts from Wikipedia that need to be removed. Mostly in this file but also three in `country.csv`, if I'm not mistaken.

File: src/data/country.csv [hunk 3]
```diff
@@ -1,312 +1,312 @@
...
+Nepal,Nepal,Nepal,Népal,Nepal,Nepál
+Palestine,Palästina,Palestina,Palestine,Palestina,Stát Palestina
+Pakistan,Pakistan,Pakistán,Pakistan,Pakistan,Pákistán
+Qatar,Katar,Catar,Qatar,Qatar,Katar
+Saudi Arabia,Saudi-Arabien,Arabia Saudita,Arabie saoudite,Saudi-Arabia,Saúdská Arábie
+Singapore,Singapur,Singapur,Singapour,Singapore,Singapur
+Syria,Syrien,Siria,Syrie,Syria,Sýrie
+Taiwan,Taiwan,Taiwán,Taïwan,Taiwan,Tchaj-wan
+Sri Lanka,Sri Lanka,Sri Lanka,Sri Lanka,Sri Lanka,Srí Lanka
+Tajikistan,Tadschikistan,Tayikistán,Tadjikistan,Tadsjikistan,Tádžikistán<br>
```
Comment (axelboc) [inline src/data/country.csv:144]: `<br>` to be removed

**Review by axelboc — APPROVED** [review]
Thanks @Flippo008 and @aplaice! Glad we can start merging in new translations!! 😍 

I've spotted a couple of problems that should be quick to fix.

File: src/data/capital.csv (hunk 2 above)
Comment (aplaice) [reply src/data/capital.csv]: I think I've removed the unnecessary disambiguation texts. (The ones for Saint-Martin are needed, just like in Spanish and French.)

**Comment by aplaice** [comment]
Thanks @axelboc for the review! I think I've fixed everything.

**Comment by axelboc** [comment]
Hope the rebase for the Russian translation won't be too dreadful @ohare93 😓

**Comment by aplaice** [comment]
Thanks @ohare93 for making this possible, in the first place, and thanks @axelboc for merging!

**Pull request merged**